src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -45,19 +46,22 @@ public BookDto getBookById(@PathVariable Long id) {
         return bookService.get(id);
     }
 
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
```

Comment:
Please, remove value from annotation under the controller
`@RequestMapping("/books")`

Reply:
not fixed